sim-cli/src/config.rs

```diff
@@ -0,0 +1,209 @@
+use std::{path::PathBuf, str::FromStr};
+
+use anyhow::Context;
+use log::LevelFilter;
+use serde::Deserialize;
+
+use crate::Cli;
+
+#[derive(Debug)]
+pub struct SimulationConfig {
+    // Path to a directory where simulation results are save
+    pub data_dir: PathBuf,
+    // Path to simulation file
+    pub sim_file: PathBuf,
+    // Duration for which the simulation should run
+    pub total_time: Option<u32>,
+    // Number of activity results to batch together before printing to csv file [min: 1]
+    pub print_batch_size: u32,
```

Comment:
Pre-existing: could you standardize to use documentation comments here?
This was updated across the repo in #160, but won't be detected since we've moved these vaues.